.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ name: Release
 
 env:
   cache-name: release-ci
-  publicecr-name: chainlink
+  publicecr-name: z0b1w9r9
```

Comment:
Did you want to swap alias over then merge this in?

Reply:
Actually it should be fine, the alias is more for external consumers than us anyway

Reply:
I mainly swapped it because we don't have the Secure SDLC alias thing figured out with AWS support yet :) The ID should work regardless of the alias, so it's probably even more future-proof (if slightly more confusing).